hw/ip/aon_timer/data/aon_timer.prj.hjson

```diff
@@ -7,8 +7,14 @@
     design_spec:        "hw/ip/aon_timer/doc",
     dv_doc:            "",
     hw_checklist:       "",
-    version:            "0.1",
-    life_stage:         "L0",
-    design_stage:       "D0",
-    verification_stage: "V0",
+    sw_checklist:       "sw/device/lib/dif/dif_aon_timer",
+    revisions: [
+      {
+        version:            "0.1",
+        life_stage:         "L0",
+        design_stage:       "D0",
+        verification_stage: "V0",
+        dif_stage:          "S0",
+      }
+    ]
```

Comment:
Can you turn this back to how it was, and just add the `dif_stage: "S0"` key, rather than creating "revisions" - revisions is for when multiple named versions exist, which we don't yet have for this hardware.

Reply:
Sure I can do, seems like a waste though - to me this falls under "Nit".